test/models/ocelot/stanzas/ChoiceStanzaSpec.scala

```diff
@@ -292,6 +292,18 @@ class ChoiceStanzaSpec extends BaseSpec {
       MoreThanOrEqualsTest("4", "hello").eval(LabelCache()) shouldBe false
     }
 
+    "provide support to LessThanTest" in {
```

Comment:
All of this family of tests compare integers with integers, strings with strings or integers with strings. Given that IHT works with currency it would be good to add some more realistic tests.